ext/native-decls/GetPlayerProofs.md

```diff
@@ -0,0 +1,27 @@
+---
+ns: CFX
+apiset: server
+---
+
+## GET_PLAYER_PROOFS
+
+```c
+void GET_PLAYER_PROOFS(Player player, int* bulletProof, int* fireProof, int* explosionProof, int* collisionProof, int* meleeProof, int* drownProof, int* steamProof);
```

Comment:
Would maybe be nice to follow the same param order like `_GET_ENTITY_PROOFS` with `(Entity entity, BOOL* bulletProof, BOOL* fireProof, BOOL* explosionProof, BOOL* collisionProof, BOOL* meleeProof, BOOL* steamProof, BOOL* p7, BOOL* drownProof)`

so switch `steamProof` and `drownProof`.